src/app/pages/ArticlePage/ArticlePage.jsx

```diff
@@ -103,6 +103,7 @@ const ArticlePage = ({ pageData }) => {
     pageData,
   );
   const recommendationsData = pathOr([], ['recommendations'], pageData);
+  const isPGL = path(['metadata', 'type'], pageData) === 'PGL';
```

Comment:
Could we use optional chaining instead of `path` here?
```suggestion
  const isPGL = pageData?.metadata?.type === 'PGL';
```